src/Config/CS_SDK.props

```diff
@@ -63,6 +63,12 @@
     <UseWPF>true</UseWPF>
     <UseWindowsForms>true</UseWindowsForms>
   </PropertyGroup>
+  <!--Temporary configuration to make core test projects load UI packages successfully -->
+  <PropertyGroup Condition=" '$(TestProjectDefaults)' == 'true' AND $(Platform.Contains('Windows')) ">
+    <TargetFramework>net6.0-windows</TargetFramework>
+    <UseWPF>true</UseWPF>
+    <UseWindowsForms>true</UseWindowsForms>
+  </PropertyGroup>
```

Comment:
dynamoCoreTests and some other projects still want to load/test UI packages.
We need to separate them out, but for now we can consider them all UI projects

Reply:
Can you explain how this change brings down the testing time?

Reply:
Less logs and less exceptions thrown during tests

Reply:
How was it working without setting `UseWpf` and `UseWinforms` to `true` before?

Reply:
What evidence suggests this: `dynamoCoreTests and some other projects still want to load/test UI packages.`, modules loaded when running dynamocoretests?

Reply:
There might be junk packages (with you refs) that are leftover from other UI tests

Reply:
Well, the evidence was in the logs/exceptions. Ex “cannot find windowsBase”